src/styles/globals.css

```
@@ -285,6 +296,20 @@
   }
 }
 
+@utility text-swap-layer {
```

Stylelint (17.11.1)

[error] Unexpected unknown at-rule "`@utility`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)